### PR TITLE
Elastic delployment automation mc

### PR DESCRIPTION
**Pull request by ianrscottexp**
Features:
- Add script to be able to activate a rule by name
- Add ability to update integration settings for any pkg policy on create
- 
Bugs:
- Add experimental flag to get_integration function to get all integrations.  This fixes issue where some integrations weren't showing up and the implementation would fail
- Update variables from integration name to integration title. This is part of the solution to bypass integration check using the integration name and version. It was found the name and title are different.

### CODE REVIEW

File: plugins/modules/elastic_detection_rule.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    else:
+        results['changed'] = True
+
+    kibana = Kibana(module)
+    if state == "present":
+      if active == True:
+        sec_rule_info = kibana.activate_security_rule(security_rule_name)
+        if sec_rule_info == security_rule_name + ': Rule is already enabled':
+          results['changed'] = False
+    results['sec_rule_info'] = sec_rule_info
```
Comment (exp-hc) [inline plugins/modules/elastic_detection_rule.py:63]: `sec_rule_info` won't be set if `state` isn't 'present' AND `active` isn't `True` but the current logic will still try to use call it.

**Review by exp-hc — APPROVED** [review]
Found one issue but it isn't critical

**Comment by ianrscottexp** [comment]
Absolutely right. I have made the update

File: plugins/modules/elastic_detection_rule.py (hunk 1 above)
Comment (ianrscottexp) [reply plugins/modules/elastic_detection_rule.py]: updated to resolve the issue.  good catch

**Review by mgaruccio — APPROVED** [review]
Made a suggestion on inputs for the new module, but I'm good with merging the current design if you decide that's the better choice here.

**Review by jesselarrabee — APPROVED** [review]

**Pull request merged**